### ISSUE TITLE
Can we make [[functionLocation]] a clickable link? (within Breakpoint Hover)

### ISSUE THREAD

**Issue by jasonwilliams** [microsoft/vscode-js-debug#1047]
Sometimes I need to go deep into library code, thanks to the debugger i'm able to find the function to investigate. However I always find it quite fiddly to go directly to this function. Below you'll see what I mean.

In this gif the path is long, and I need to keep it selected whilst moving the horizontal scroll bar. It can become a right chore.
![functionLocation](https://user-images.githubusercontent.com/936006/124036634-38058480-d9f6-11eb-90c3-56b12bdb11bf.gif)

Is it possible to make that path a clickable link which opens a new tab?

**Comment by connor4312** [comment]
@isidorn I think we need some new way to indicate clickability or some link detection here, no?

**Comment by isidorn** [comment]
@connor4312 yeah. I think this would ideally work by doing link detection in the debug hover. The same way we detect links in the debug console.
Code pointer for link detector https://github.com/microsoft/vscode/blob/main/src/vs/workbench/contrib/debug/browser/linkDetector.ts#L40

However I am not super happy with the link detector. Though I see it has already spread across the workbench. So maybe we should just start using it in the debugHover.
